go-controller/pkg/kubevirt/pod.go

```diff
@@ -1,6 +1,55 @@
 package kubevirt
 
+import (
+	"context"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// AllowPodBridgeNetworkLiveMigration will return true if the pod belongs
+// to kubevirt and should use the live migration features
 func AllowPodBridgeNetworkLiveMigration(annotations map[string]string) bool {
 	_, ok := annotations[AllowPodBridgeNetworkLiveMigrationAnnotation]
 	return ok
 }
+
+// FindPodsByVMLabel will return pods belong to the same vm annotated at pod
+func FindPodsByVMLabel(client clientset.Interface, pod *corev1.Pod) ([]corev1.Pod, error) {
```

Comment:
`FindAllPodsForVM` ? `FindAllVMPods` ? The label is basically an implementation detail.

And should we really use the client set ? Shouldn't we get this info from the lister ? 

Finally, would it make sense for the func to receive as parameter the VM name ? Or UUID (I don't recall right now what data is stored in this label).

Reply:
The idea is to return the virt-launcher pods related to another virt-launcher pods, will change the name and use the watchFactory